src/app/store/shortcuts.js

```diff
@@ -777,4 +778,40 @@ export const shortcuts = [
         isActive: true,
         category: TOOLBAR_CATEGORY,
     },
+    { // Jog A+
+        id: 71,
+        title: 'Jog: A+',
+        keys: ['ctrl', '6'].join('+'),
+        cmd: 'JOG',
+        payload: {
+            axis: { [AXIS_A]: 1 },
+        },
+        preventDefault: false,
+        isActive: true,
+        category: JOGGING_CATEGORY,
+    },
+    { // Jog A-
+        id: 72,
+        title: 'Jog: A-',
+        keys: ['ctrl', '4'].join('+'),
+        cmd: 'JOG',
+        payload: {
+            axis: { [AXIS_A]: -1 },
+        },
+        preventDefault: false,
+        isActive: true,
+        category: JOGGING_CATEGORY,
+    },
```

Comment:
These shortcuts need to be in the /app/JogControl/index.js, hence why they are having trouble rendering to the list of shortcuts. The logic for jogging using the shortcuts is also there, so you can use it to check if the a-axis values are being sent to the controller.

Also when adding new shortcuts, make sure the id is unique, i believe a few of them used here are already taken with other shortcuts. We'll probably change the way ids are set in the future but for now they need to be unique manually